### PR TITLE
app-emulation/libvirt: Specify docdir on configure cmd line

### PR DESCRIPTION
**Pull request by zippy2**
Up until very recently libvirt did not allow specifying docdir
via any (meson) configure option. That is why we patched
meson.build and changed docdir to point where we wanted it to
point. Well, this is changed with upstream commit of
v7.1.0-145-gee4a392dda. Therefore, don't patch the meson.build
file anymore and specify docdir like this:

  -Ddocdir="${EPREFIX}/usr/share/doc/${P}"

Signed-off-by: Michal Privoznik <mprivozn@redhat.com>

### CODE REVIEW

**Comment by zippy2** [comment]
cc @tamiko @trofi

File: app-emulation/libvirt/libvirt-9999.ebuild
Comment (juippis) [inline app-emulation/libvirt/libvirt-9999.ebuild:278]: We use `${PF}` for our docdirs. LGTM otherwise.

**Review by tamiko — APPROVED** [review]

File: app-emulation/libvirt/libvirt-9999.ebuild
Comment (zippy2) [reply app-emulation/libvirt/libvirt-9999.ebuild]: Pushed fixed version.

**Review by juippis — APPROVED** [review]
Thanks for going the extramile and fixing the dir, we could've easily do that while merging. I was waiting for feedback from other maintainer, but all the better now!

**Pull request closed without merging**
